code/ui/.storybook/main.ts

```diff
@@ -3,14 +3,17 @@ import type { StorybookConfig } from '../../frameworks/react-vite/dist';
 const config: StorybookConfig = {
   stories: [
     {
-      directory: '../manager/src/**/',
-      files: '*.stories.@(ts|tsx|js|jsx|mdx)',
-      titlePrefix: 'Manager',
+      directory: '../manager/src',
+      // these titles uses a special unicode character for the slash as a workaround for https://github.com/storybookjs/storybook/issues/9761
```

Comment:
This is hacky, and if anyone has strong opinions against this let me know.
The initial setup just names them by their directories but that became rather confusing because we have `components`, but also components in `blocks` and `ui`.
So this approach is more explicit.